org.mixedrealitytoolkit.spatialmanipulation/BoundsControl/BoundsHandleInteractable.cs

```diff
@@ -117,27 +134,18 @@ public void Start()
         /// </summary>
         protected virtual void LateUpdate()
         {
-            // Do our IsOccluded "setter" in Update so we don't do this multiple times a frame.
-            if (IsOccluded != wasOccludedLastFrame)
-            {
-                wasOccludedLastFrame = IsOccluded;
-                if (handleRenderer != null)
-                {
-                    handleRenderer.enabled = !IsOccluded;
-                }
-                colliders[0].enabled = !IsOccluded;
-            }
-
             // Maintain the aspect ratio/proportion of the handles, globally.
             transform.localScale = Vector3.one;
-            transform.localScale = new Vector3(1.0f / transform.lossyScale.x,
-                                               1.0f / transform.lossyScale.y,
-                                               1.0f / transform.lossyScale.z);
-            
+            transform.localScale = new Vector3(1.0f / (transform.lossyScale.x == 0 ? 1 : transform.lossyScale.x),
+                                1.0f / (transform.lossyScale.y == 0 ? 1 : transform.lossyScale.y),
```

Comment:
Never mind I see `SqueezeBoxVisuals::OnEnable` is calling this.

However, I think this could be causing part of your bug.  Since `SqueezeBoxVisuals::OnEnable` and  `BoundsHandleInteractable::Awake" calls `HideOnStartup()`, and `HideOnStartup()` sets wasOccludedLastFrame to true. LateUpdate() incorrectly using the IsOccluded value too soon.  

I think we need to re-think `HideOnStartup` that was added with this PR....https://github.com/microsoft/MixedRealityToolkit-Unity/pull/11698

Reply:
If we need to keep HideOnStartup, then should set IsOccluded to true in HideOnStartup...that should fix stuff